window.guardian.d.ts

```diff
@@ -35,6 +35,7 @@ declare global {
         GoogleAnalyticsObject: string;
         ga: UniversalAnalytics.ga;
         initAutomatJs?: Function;
+        __import__: (url: string) => Promise<any>;
```

Comment:
Can we namespace the import if that's the case to something like `window.guardian.functions.import`? 

Probably needs thought if external modules are going to rely on it being available on the window in the future, so refactoring out at a later stage will be difficult.

Reply:
Note: This the reason for `request changes` just because this is not easily back-out-able in the long term.